containers/daap-resync-unprocessed-files/src/var/task/resync_unprocessed_files.py

```diff
@@ -16,77 +24,62 @@
     "ACL": "bucket-owner-full-control",
     "ServerSideEncryption": "AES256",
 }
-raw_data_bucket = os.environ.get("RAW_DATA_BUCKET", "")
-curated_data_bucket = os.environ.get("CURATED_DATA_BUCKET", "")
-log_bucket = os.environ.get("RAW_DATA_BUCKET", "")
+
+raw_data_bucket = get_raw_data_bucket()
+curated_data_bucket = get_curated_data_bucket()
+log_bucket = get_log_bucket()
 athena_load_lambda = os.environ.get("ATHENA_LOAD_LAMBDA", "")
 
 
 def handler(event, context):
     data_product_to_recreate = event.get("data_product", "")
-    raw_prefix = f"raw_data/{data_product_to_recreate}"
-    curated_prefix = f"curated_data/database_name={data_product_to_recreate}"
+
+    data_product = DataProductConfig(
+        name=data_product_to_recreate,
+        raw_data_bucket=raw_data_bucket,
```

Comment:
you can leave these bucket params out if you want, I made it default to the environment variables